kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -611,7 +611,6 @@ impl<'tcx> GotocCtx<'tcx> {
                 loc,
             ),
             "simd_xor" => codegen_intrinsic_binop!(bitxor),
-            "size_of" => codegen_intrinsic_const!(),
```

Comment:
Are we sure this is no longer possible (it still exists in `core`:  https://doc.rust-lang.org/src/core/intrinsics.rs.html#887). If so, I think it's better if we replace this with `unreachable!` similar to what we do for `copy_nonoverlapping`: https://github.com/model-checking/kani/blob/d9cf03a2ac1bf80f339ebec2edf465a43108f735/kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs#L469 so that we get a panic at compile time as opposed to verification failure.

Reply:
Good call, done!